pkg/apis/config/feature_flags.go

```diff
@@ -113,11 +114,16 @@ type FeatureFlags struct {
 	SendCloudEventsForRuns           bool
 	AwaitSidecarReadiness            bool
 	EnforceNonfalsifiability         string
-	ResourceVerificationMode         string
-	EnableProvenanceInStatus         bool
-	ResultExtractionMethod           string
-	MaxResultSize                    int
-	CustomTaskVersion                string
+	// VerificationNoMatchPolicy is the feature flag for "trusted-resources-verification-no-match-policy"
```

Comment:
nit: this is already pretty well documented; it doesn't really need to be documented here; up to you